packages/generator-single-spa/src/react/templates/.ts.eslintrc

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": [
+    "ts-react-important-stuff",
+    "plugin:prettier/recommended"
+  ],
+  "parser": "@typescript-eslint/parser"
```

Comment:
Why not babel-eslint here? babel-eslint@next supports using your own babelrc, which includes `@babel/preset-typescript`

Reply:
Oh yeah, that makes sense! I was just copying from another repo.